style/app.scss

```diff
@@ -654,68 +657,110 @@ a:active
 
 #fce
 {
-  height : 40px;
-  width  : 100%;
-  border : solid 1px #000000;
+  height      : 40px;
+  width       : 140px;
+  padding-top : 20px;
 }
 
 #fcecount
 {
-  width            : 53%;
+  border-radius    : 10px;
   height           : 40px;
+  width            : 140px;
   float            : left;
-  background-color : #9c91b0;
+  background-color : #ffffff;
+  padding-top      : 8px;
   text-align       : center;
-  padding-left     : 15px;
-  padding-top      : 5px;
-  font-size        : 18px;
+  font-size        : 16px;
 }
 
-#reset
+.label-hidden
 {
-  text-align : center;
-  float      : left;
-  width      : 47%;
-  height     : 40px;
-  border     : solid 1px #000000;
-  cursor     : pointer;
+  clip        : rect(0 0 0 0);
+  clip-path   : inset(50%);
+  height      : 1px;
+  overflow    : hidden;
+  position    : absolute;
+  white-space : nowrap;
+  width       : 1px;
 }
 
-#reset:hover
+#reset-selections
 {
-  background-color : #222222;
-  color            : #ffffff;
+  margin           : 10px;
+  text-align       : center;
+  float            : center;
+  width            : 90px;
+  height           : 20px;
+  cursor           : pointer;
+  border-radius    : 15px;
+  font-size        : 10px;
+  background-color : #ffffff;
+  color            : #c8afce;
+  border           : #e8e9eb;
+  box-shadow: 3px 3px 3px #e8e9eb;
+  outline          : none;
+}
+
+#reset-selections:hover
+{
+  background-color : #cac4d4;
 }
 
 #container
 {
-  height   : calc(100% - 50px);
-  width    : 100%;
-  position : relative;
-  background: #f0f5f8;
+  height     : calc(100% - 50px);
+  width      : 100%;
+  position   : relative;
+  background : #f0f5f8;
 }
 
 #sidebar
 {
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
+  border-radius    : 20px;
   float            : left;
-  background-color : #7a6a96;
   position         : absolute;
-  padding-left     : 23px;
 }
 
 #sidebar-button
 {
   cursor           : pointer;
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
+  border-radius    : 10px;
+  width            : 30px;
+  height           : 15px;
   float            : left;
-  background-color : #5c497e;
+  background-color : #d9d9d9;
   position         : absolute;
-  border           : solid 1px #000000;
+  margin-top       : 10px;
+  margin-left      : 60px;
+  text-align       : center;
+  z-index          : 2;
+}
+
+#selected-courses
+{
+  padding-top: 10px;
+  font-size        : small;
+  text-decoration  : underline;
+  color            : #a47cb8;
+  background-color : none;
```

Comment:
Reminder that you can remove all `background-color` rules for the individual elements once you've implemented my above comment.